spec/support/date_time_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module DateTimeHelpers
+  module_function
+
+  def travel_to_days_later(days)
```

Comment:
`travel_to(2.days.ago)` doesn't work?

Reply:
No, `2.days.ago` is specific to Rails, which we don't have here.